README.md

```diff
@@ -44,5 +44,5 @@ Make sure the build completed successfully.
 Following the above file structure, copy the generated `.jar` file located at `eip-client/app/target/` path into the working root directory and run the app on commandline with the following command.
 
 ```
-java -jar eip-client-app-1.0.0-SNAPSHOT.jar
+java -jar client-app-1.0.0-SNAPSHOT.jar
```

Comment:
@rbuisson don't you think that the JAR itself should be named like `openmrs-eip-client-<version>.jar`?

Reply:
@rbuisson, simply pinging you here....

Reply:
@Ruhanga I think you can go ahead with my suggestion.

Reply:
Ok, thanks.